english/109_jumla.md

```diff
@@ -493,23 +501,23 @@ The same particle is used with with personal pronouns too.
 ### Tense and Aspect Particles
 
 In Pandunia, tense can be expressed with time words and time phrases if needed.
-The general time words are **ceno** (past), **zayo** (present) and **poso** (future, upcoming).
+The general time words are **pas** (past), **zai** (present) and **pos** (future, upcoming).
```

Comment:
**pas** and **pos** are too similar but both have quite good support from source languages. I want to pair **pos** ('after') with **chen** ('before') and **pas** ('past', 'bygone') with **vil** ('future') or another word.

Reply:
yeah, I did notice that.  so in this seccion, should the basick tenses be **chen** and **pos**, or **pas** and **vil**?

Reply:
**pas** and **vil**.